scripts/portstat

```diff
@@ -149,20 +161,57 @@ class Portstat(object):
                               data.rx_drop, data.rx_ovr,
                               data.tx_ok, STATUS_NA, STATUS_NA, data.tx_err,
                               data.tx_drop, data.tx_ovr))
-
-
         if use_json:
             print table_as_json(table, header_all if print_all else header)
         else:
             print tabulate(table, header_all if print_all else header, tablefmt='simple', stralign='right')
 
+    def ne_cnstat_print(self, cnstat_dict, use_json, print_all):
+        """
+            Print the network errors cnstat.
+        """
+        table = []
+        for key, data in cnstat_dict.iteritems():
+            if key == 'time':
+                continue
+            if print_all:
+                table.append((key, self.get_port_state(key),
+                              data.rx_err, data.rx_drop, data.rx_ovr,
+                              data.tx_err, data.tx_drop, data.tx_ovr))
+            else:
+                table.append((key, self.get_port_state(key),
+                              data.rx_err, data.rx_drop, data.rx_ovr,
+                              data.tx_err, data.tx_drop, data.tx_ovr))
+        if use_json:
+            print table_as_json(table, ne_header_all if print_all else ne_header)
+        else:
+            print tabulate(table, ne_header_all if print_all else ne_header, tablefmt='simple', stralign='right')
+    def nr_cnstat_print(self, cnstat_dict, use_json, print_all):
```

Comment:
Function name is not clear. What does `nr_` prefix mean?

Reply:
Updated function name to be network_rate_cnstat_print.